tox.ini

```diff
@@ -32,7 +33,7 @@ allowlist_externals =
     sh
 
 [testenv:linters]
-basepython = python3.10
+basepython = python3.9
```

Comment:
linters should run with `python3.10`

Reply:
https://github.com/wazo-platform/wazo-amid/pull/89